eyieko/landville-frontend
# #165880984 Retrieve Client Reviews

## Description ##
Retrieve all reviews on a client company

## Type of change ##
Please select the relevant option

## How Has This Been Tested? ##
Please describe the tests that you ran to verify your changes

## Checklist: ##

## How can this PR be manually tested: ##
- navigate to the property details page i.e `/properties/slug`
- You should be able to get the latest Review on the client.
- Click `More Reviews` button (ie if that company has any reviews) which should redirect you to the page (Client Details Page)
  where you can see all the rest of the reviews when you scroll through the list of reviews.
- [Review app](https://landville-frontend-pr-47.herokuapp.com)


## Related Pivotal Tracker stories ##
[#165880984](https://www.pivotaltracker.com/story/show/165880984)
